src/components/ChartComponent.js

```diff
@@ -19,8 +19,8 @@ class ChartComponent extends Component {
   }
 
   renderChart(measurements) {
-    const data = measurements.map(measurement => measurement['Wasted time (ms)']);
-    const labels = measurements.map(measurement => measurement['Owner > component']);
+    const data = measurements.map(measurement => measurement.inclusiveRenderDuration);
+    const labels = measurements.map(measurement => measuremen.key);
```

Comment:
I think you made a typo here. Could you resolve this.
Once tests pass I will merge :)